newIDE/app/src/GameDashboard/Widgets/AnalyticsWidget.js

```diff
@@ -119,7 +119,7 @@ const AnalyticsWidget = ({ game, onSeeAll, gameMetrics, gameUrl }: Props) => {
               ) : (
                 <Column expand noMargin>
                   <Line alignItems="center" justifyContent="space-between">
-                    <Text size="block-title" noMargin>
+                    <Text size="sub-title" noMargin>
```

Comment:
What does this change do?

Reply:
This change a title to get a consistent UI with the other sub titles. Look at "Sessions" before/after.
![image](https://github.com/user-attachments/assets/940787f4-48d2-4b25-b286-5c63460a7028)